### ISSUE TITLE
Tuning Neural Networks with Keras Tuner

### ISSUE THREAD

**Issue by Cgarg9** [PEC-CSS/AI#41]
### Description:
Use Keras Tuner to find the best hyperparameters for a CNN on the MNIST dataset.

### Tasks:

- Load and preprocess the MNIST dataset.
- Define a CNN architecture using TensorFlow/Keras.
- Use Keras Tuner to optimize learning rate, number of filters, and dropout rate.
- Compare training loss before and after tuning.
- Name the notebook keras_tuner_mnist.ipynb.
- Update the README file with relevant references.

**Comment by guntaaskapoor24** [comment]
@Cgarg9 Please assign this issue to me

**Comment by AekamSidhu** [comment]
Hi can i be assigned this issue @Cgarg9

**Comment by Cgarg9** [comment]
@AekamSidhu need to close off the last issue or i need to remove you from that

**Comment by AekamSidhu** [comment]
HI can i be assigned this now? @Cgarg9

**Comment by Cgarg9** [comment]
ok.

**Comment by Cgarg9** [comment]
@AekamSidhu please try to prepare a small assignment on this as well.

**Comment by Cgarg9** [comment]
@AekamSidhu updates?

**Comment by Cgarg9** [comment]
@AekamSidhu will unassign by tonight if no updates.

**Comment by Kanavpreet-Singh** [comment]
@Cgarg9 can i work on this issue?

**Comment by Cgarg9** [comment]
sure
